packages/dashboard/src/components/MantineGlobal.tsx

```diff
@@ -42,6 +48,33 @@ function MantineGlobal(): JSX.Element {
             [`${truffleBgColorSelectors}, ${truffleOffBgColorSelectors}`]: {
               transition: "background-color 0.1s"
             },
+            ".trfl-AppShell-main": {
+              width: "auto"
+            },
+            // class for highlighting source material in debugger
+            ".truffle-debugger-text-highlighted": {
+              backgroundColor:
+                colorScheme === "dark" ? "#4444aa60" : "#ffff0050"
+            },
+
+            ".truffle-debugger-input-and-button": {
+              display: "flex",
+              flexGrow: 1
+            },
+            ".truffle-debugger-input-and-button > div > input": {
+              borderTopRightRadius: 0,
+              borderBottomRightRadius: 0,
+              border: 0,
+              height: 42,
+              backgroundColor:
+                theme.colorScheme === "dark"
+                  ? theme.colors["truffle-brown"][5]
+                  : "#FFFFFF"
+            },
```

Comment:
I moved ".truffle-debugger-input-and-button" and the class below into the component file. The ".truffle-debugger-text-highlighted" is currently global because those classes are attached in the utilities before they get to the component. Should we architect this differently?